src/main/java/com/carmen/app/control/CarService.java

```diff
@@ -1,62 +1,61 @@
 package com.carmen.app.control;
 
 import java.util.List;
-import java.util.UUID;
+//import java.util.UUID;
 
 import javax.ejb.Stateless;
+import javax.interceptor.Interceptors;
 import javax.persistence.EntityManager;
 import javax.persistence.EntityManagerFactory;
 import javax.persistence.Persistence;
 import javax.persistence.TypedQuery;
 
-import org.apache.log4j.LogManager;
-import org.apache.log4j.Logger;
+//import org.apache.log4j.LogManager;
+//import org.apache.log4j.Logger;
 
 import com.carmen.app.entities.Car;
 import com.carmen.app.exceptions.CarNotFoundException;
+import com.carmen.app.utils.LoggInterceptor;
 
 @Stateless
+@Interceptors(LoggInterceptor.class)
```

Comment:
This way of registering a interceptor it works, but the usual way is via annotations. Could you please switch to the annotation approach, please ?

https://javaee.github.io/tutorial/interceptors002.html
**Binding Interceptors to Components**